internal/cli/main.go

```diff
@@ -27,9 +27,8 @@ const (
 	gotoUsage = `goto V       Migrate to version V`
 	upUsage   = `up [N]       Apply all or N up migrations`
 	downUsage = `down [N]     Apply all or N down migrations`
```

Comment:
Move the `-all` flag docs to the `down` command

Reply:
@dhui Updated my commit :smile: